holoviews/core/data/__init__.py

```diff
@@ -8,8 +8,12 @@
 import numpy as np
 import param
 
-from ..dimension import redim
-from ..util import unique_iterator
+from ..dimension import redim, Dimension, process_dimensions
+from ..element import Element
+from ..ndmapping import OrderedDict
+from ..spaces import HoloMap, DynamicMap
+from ..util import (basestring, dimension_range as d_range, get_param_values,
+                    isfinite, process_ellipses, unique_iterator, wrap_tuple)
```

Comment:
I had some weird issues when importing ``from .. import util`` getting the wrong utilities, hence I did this.

Reply:
I see. I would prefer to figure out the issue rather than switching to the unqualified version...